fixed argument of SetUnitsOfMeasurement

YoshitoMomiyama opened the pull request:
modify argument of SetUnitsOfMeasurement from unitsOfMeasurement to unitsOfMeasurementList

YoshitoMomiyama committed:
fixed argument of SetUnitsOfMeasurement

modify argument of SetUnitsOfMeasurement from unitsOfMeasurement to unitsOfMeasurementList

asanoaozora commented:
You're right, there's a mistake for the name of the parameter, but if we change it, it makes also sense to change the argument of GetUnitsOfMeasurement from unitsOfMeasurement to unitsOfMeasurementList.
Could you please update your pull request accordingly ? I'll merge afterwards.

YoshitoMomiyama committed:
fixed argument of GetUnitsOfMeasurement

modify argument of GetUnitsOfMeasurement from unitsOfMeasurement to unitsOfMeasurementList

YoshitoMomiyama commented:
Thank you for review.
I also modified the argument of GetUnitsOfMeasurement.
